### PR TITLE
New Charts Page

### PR DESCRIPTION
**Pull request by Elkrival**
Add the New Charts Page.

### CODE REVIEW

File: server/templates/NewChart.template.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import basePathConfig from '../configs/basePathConfig'
+
+const basePath = basePathConfig || ''
+
+export default () => {
+	return `
```
Comment (mikestone14) [inline server/templates/NewChart.template.js]: can you fix the formatting of this file?

File: views/Chart.react.js [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
   return (
-    <div>
+    <>
       <h1> Charts</h1>
-    </div>
+      <br />
+      <Button
+        variant="outlined"
+        color="primary"
+        href={`${basePath}/charts/new`}
```
Comment (mikestone14) [inline views/Chart.react.js]: can we extract the routes to a routes object so we can reference this with `routes.newChart`?

File: views/Chart.react.js [hunk 3]
```diff
@@ -1,9 +1,21 @@
...
+
+const basePath = basePathConfig || '';
 
 export default function Charts() {
   return (
-    <div>
+    <>
       <h1> Charts</h1>
-    </div>
+      <br />
```
Comment (mikestone14) [inline views/Chart.react.js]: do we need this break tag? Can we use css instead?

File: views/Chart.react.js (hunk 2 above)
Comment (Elkrival) [reply views/Chart.react.js]: [89c25f1](https://github.com/AMP-SCZ/dpdash/pull/78/commits/89c25f103294e3e583dd2ca8d95164bd04b81388)

File: views/Chart.react.js (hunk 3 above)
Comment (Elkrival) [reply views/Chart.react.js]: [89c25f1](https://github.com/AMP-SCZ/dpdash/pull/78/commits/89c25f103294e3e583dd2ca8d95164bd04b81388)

File: server/templates/NewChart.template.js (hunk 1 above)
Comment (Elkrival) [reply server/templates/NewChart.template.js]: [a826d0d](https://github.com/AMP-SCZ/dpdash/pull/78/commits/a826d0d8b81c3e20be808d5bd4546eff18343fff)

**Review by mikestone14 — APPROVED** [review]

**Review by mikestone14 — APPROVED** [review]

**Pull request merged**